content-src/lib/selectLayoutRender.js

```diff
@@ -139,19 +139,29 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
     return { ...component, data };
   };
 
+  const filterComponent = (c) => {
+    return (
+      !filterArray.includes(c.type) &&
+      (!c.sponsored || spocs.showSpocs)
```

Comment:
ah I see you have  sponsored property to indicate a campaign is paid promotion.

Reply:
Yeah, I think one bit that's confusing is anything that comes from the old spocs endpoint, doesn't have the `sponsored: true` because it explicitly came from a sponsored endpoint so we can assume it's all spocs.

same is true for any feed data coming from a card grid in a sponsored collection. We can assume that's all sponsored content.

Reply:
is it safe to assume any feed data coming from cardgrid in a sponsored collection, will have context property set , thats used in DSCard UI to display 'Sponsored by ...' label. Just want to highlight that my WIP Card UI variant patch looks for context property against an item and doesn't handle sponsored:true set against a component